README.md

```diff
@@ -0,0 +1,77 @@
+This is a dashboard for tracking outstanding bugs in **Ironic**.
+
+**Ironic** is one of the software components in OpenStack that fully manages bare-metal infrastructure; it discovers bare-metal nodes; catalogs them in a management database and manages the entire server lifecycle; from enrolling to provisioning, maintenance and decomissioning.
+
+And **OpenStack** is an opensource cloud software project that is made up of a collection of software components that provide common services for cloud infrastructure.
```

Comment:
remove "And" at the beginning
s/"that is made up"/""that includes"
s/"that provide"/"providing"

Reply:
maybe also add a link to the openstack main page? https://www.openstack.org/